packages/components/psammead-media-indicator/src/__snapshots__/index.test.jsx.snap

```diff
@@ -396,6 +401,7 @@ exports[`MediaIndicator should render video indicator correctly when inline on R
 <div
   aria-hidden="true"
   class="c0"
+  dir="ltr"
```

Comment:
This should be "rtl" since it is for the test `MediaIndicator should render video indicator correctly when inline on RTL 1`. It's getting the default value because `dir='rtl'` is not passed to that test

Reply:
True, forgot to pass it also in the tests 👍